src/test/java/FlightsReporterTests.groovy

```diff
@@ -1,48 +1,69 @@
 import org.junit.Test
-import org.junit.Assert
-import org.junit.jupiter.api.BeforeAll
 
-import java.nio.channels.FileLock
 import java.time.LocalDateTime
-import java.time.LocalTime;
-//import FlightsReporter
 
-import static  org.junit.jupiter.api.Assertions.*;
+
+import static org.junit.jupiter.api.Assertions.*;
 
 class FlightsReporterTests {
-    List<Flight> flights = []
+    FlightsReporter flightsReporter
 
     @Test
     void testFlightLengthInMinutes() {
-        Flight fl1 = new Flight(id: "SOFLON11", from: FlightsReporter.Airports.SOFIA, destination: FlightsReporter.Airports.LONDON,
-            departure: LocalDateTime.of(2021, 8, 3, 10, 30),
+        Flight fl1 = new Flight(id: "SOFLON11", from: Airports.SOFIA, destination: Airports.LONDON,
+                departure: LocalDateTime.of(2021, 8, 3, 10, 30),
                 arrival: LocalDateTime.of(2021, 8, 3, 13, 15),
-                passengers: 100, capacity: FlightsReporter.CAPACITIES[3], runningLate: false)
+                passengers: 100, capacity: FlightGenerator.CAPACITIES[3], runningLate: false)
 
-        assertEquals(FlightsReporter.flightLengthInMinutes(fl1), 165)
+        assertEquals(fl1.flightLengthInMinutes(), 165)
 
     }
 
     @Test
-    void testGetAllFlightsTo() {
-        FlightsReporter.printFlights(flights)
-        assert FlightsReporter.getAllFlightsTo(flights, FlightsReporter.Airports.AMSTERDAM)
-                == [flights[0], flights[1]]
-
-        assertEquals(FlightsReporter.getAllFlightsTo(flights, FlightsReporter.Airports.SOFIA),
-                Arrays.asList(flights[8], flights[9]))
-
-        assertEquals(FlightsReporter.getAllFlightsTo(flights, FlightsReporter.Airports.LONDON),
-                Arrays.asList(flights[2], flights[4], flights[7]))
-
-        assertNotEquals(FlightsReporter.getAllFlightsTo(flights, FlightsReporter.Airports.NEW_YORK),
-                Arrays.asList(flights[5], flights[6], flights[7]))
+    void getAllFlightsTo() {
+        List<Flights> flightsToAmsterdam = []
+        FlightsReporter reporter = new FlightsReporter(flights: new Flights(flights: flightsToAmsterdam))
+        assertEquals(reporter.getAllFlightsTo(Airports.AMSTERDAM), [])
+        flightsToAmsterdam = [new Flight(id: "1",
+                from: Airports.PARIS,
+                destination: Airports.AMSTERDAM,
+                departure: LocalDateTime.of(2021, 8, 1, 10, 0),
+                arrival: LocalDateTime.of(2021, 8, 1, 11, 10),
+                passengers: 150, capacity: 150, runningLate: true),
+                               new Flight(id: "2",
+                                       from: Airports.SOFIA,
+                                       destination: Airports.AMSTERDAM,
+                                       departure: LocalDateTime.of(2021, 8, 1, 8, 35),
+                                       arrival: LocalDateTime.of(2021, 8, 1, 11, 05),
+                                       passengers: 160, capacity: 180, runningLate: false)]
```

Comment:
You can use the generator and just change the "to" field for this test. You don't need all the fields for this particular reporting method